plugins/importexport/native/filter/NativeXmlSubmissionFilter.php

```diff
@@ -104,6 +106,9 @@ public function handleElement($node)
         $submission = Repo::submission()->get($submission->getId());
 
         $deployment->addImportedRootEntity(Application::ASSOC_TYPE_SUBMISSION, $submission);
+        //add Empty review stage for reviewed  submissions
+        $stageName = $node->getAttribute('stage');
```

Comment:
How about having this a few lines above, before the line number 88, so that there are no double `$node->getAttribute('stage')`?

Reply:
@bozana  sure, it was moved.